.github/actions/deploy-klt-on-cluster/action.yml

```diff
@@ -0,0 +1,39 @@
+name: "Deploy Keptn Lifecycle Toolkit on GH cluster"
+description: "Creates a Kind cluster and deploys Keptn Lifecycle Toolkit"
+inputs:
+   cert-manager-version:
+     required: true
+     description: "Version of cert-manager that should be deployed"
+runs:
+  using: "composite"
+  steps:
+  - name: Set up Go 1.x
+    uses: actions/setup-go@v3
+    with:
+      go-version: ${{ env.GO_VERSION }}
+      cache: true
+      cache-dependency-path: '**/go.sum'
+
+  - name: Download manifests
+    uses: actions/download-artifact@v3
+    with:
+      path: ~/download/artifacts
+
+  - name: Create k8s Kind Cluster
+    uses: helm/kind-action@v1.3.0
+  
+  - name: Install cert-manager
+    shell: bash
```

Comment:
do you even need bash here?

Reply:
The default shell needs to be provided in composite actions.